src/ImageRunModal.jsx

```diff
@@ -119,18 +120,37 @@ const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount }) =>
         </Grid>
     );
 
+const handleEnvValue = (key, value, idx, onChange, additem, itemCount) => {
+    // Allow the input of KEY=VALUE seperated value pairs for bulk import
+    if (value.includes('=')) {
+        const parts = value.trim().split(" ");
+        let index = idx;
+        for (const part of parts) {
+            const [envKey, envVar] = part.split('=', 2);
```

Comment:
Can you please check if both values are defined? If for example you would have empty line in between items or spaces after item, it leads to many empty items. We should skip those.
![Screenshot from 2021-12-10 12-52-43](https://user-images.githubusercontent.com/12330670/145570617-e7c870bf-74eb-4141-8d7e-89c240f1bd1b.png)
![Screenshot from 2021-12-10 12-52-06](https://user-images.githubusercontent.com/12330670/145570621-cb532ea9-dd9a-4852-9903-0775cbce99d9.png)